test/integration/scan_test.go

```diff
@@ -1972,3 +1972,17 @@ func TestCreateAsyncScan_CallExportServiceBeforeScanFinishWithRetry_Success(t *t
 	asserts.Nil(t, err)
 	assert.Assert(t, exportRes != nil, "Export response should not be nil")
 }
+
+func TestCreateScanWithResubmitFlag_ProjectNotExist_ScanCreatedSuccessfullyWithDefaultConfig(t *testing.T) {
```

Comment:
can we some how assert the config to check it is the default configs?

Reply:
I can probably create unit test for this function that will check that the config that got returned is empty

Reply:
Added unit test